app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,50 @@
+class ReviewsController < ApplicationController
+  def new
+    @game = Game.find(params[:game_id])
+    @review = Review.new
+  end
+
+  def create
+    @game = Game.find(params[:game_id])
+    @review = Review.new(review_params)
+    @review.user_id = current_user.id
+    @review.game_id = params[:game_id]
+
+    if @review.save
+      flash[:notice] = "review submitted."
+      redirect_to game_path(@game)
+    else
+      flash[:notice] = "failed to submit"
+      render :new
+    end
+  end
+
+  def edit
+    @game = Game.find(params[:game_id])
+    @review = Review.find(params[:id])
```

Comment:
You don't need to nest the `edit` or `update` actions since you have access to the review ID:

```ruby
def edit
  @review = Review.find(params[:id])
  # if you need the game, you can retrieve it via the review
  @game = @review.game
end
```